include/mbedtls/ssl_internal.h

```diff
@@ -1282,7 +1282,10 @@ int mbedtls_ssl_read_record( mbedtls_ssl_context *ssl,
 
 int mbedtls_ssl_fetch_input(mbedtls_ssl_context* ssl, size_t nb_want);
 int mbedtls_ssl_flush_output(mbedtls_ssl_context* ssl);
+
+#if !defined(MBEDTLS_SSL_US_EMPS)
```

Comment:
[mbedtls_ssl_write_record
](https://github.com/hannestschofenig/mbedtls/blob/tls13-prototype/library/ssl_tls13_generic.c#L840) is still used in non-MPS code. 

I got compiler error when I enabled the `MBEDTLS_SSL_TLS13_COMPATIBILITY_MODE`
```
../../library/libmbedtls.a(ssl_tls13_generic.c.o): In function `mbedtls_ssl_write_change_cipher_spec_process':
/home/lhuang04/upstream/library/ssl_tls13_generic.c:840: undefined reference to `mbedtls_ssl_write_record'
```

And [here](https://github.com/hannestschofenig/mbedtls/blob/tls13-prototype/library/ssl_tls13_generic.c#L1034) is another one.

Reply:
This should be fixed now.